app/src/main/java/com/framgia/chat_03/Utils.java

```diff
@@ -0,0 +1,7 @@
+package com.framgia.chat_03;
+
+public class Utils {
```

Comment:
nếu đây là sharepreference key thì em nên đặt tên là 
```suggestion
public class SharePreferenceUtils {
```